src/identity-providers/add/DetailSettings.tsx

```diff
@@ -106,6 +109,19 @@ export const DetailSettings = () => {
   const history = useHistory();
   const { realm } = useRealm();
 
+  const MapperLink = ({ name, mapperId }: IdPWithMapperAttributes) => (
+    <Link
+      to={toIdentityProviderEditMapper({
+        realm,
+        alias: alias!,
```

Comment:
The `alias` prop is of type `string` and thus cannot be `undefined`, this means the non-null assertion can be removed.
```suggestion
        alias,
```